package.json

```diff
@@ -28,9 +28,6 @@
     "email": "ismaelga@gmail.com"
   },
   "main": "dist/react-json-editor.js",
-  "peerDependencies": {
-    "react": ">=0.12.2 <1.0.0"
-  },
```

Comment:
@jahed Ack on the peer dependency. Whats the status of this? I'm from the https://github.com/lumc-nested/nested-editor team as referenced above and we would like to move to React 15.